scripts/spelling.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Library General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# spellintian.sh
+# Copyright (C) 2023 Perry Naseck, Peter Newman
+
+# This script is based on Travis CI tests by Peter Newman
+current_test=""
+# run_single_only=false
```

Comment:
?

Reply:
Oops that wasn't supposed to be left in there!

Reply:
Fixed!